ios-training/Model/WeatherModel.swift

```diff
@@ -11,10 +11,32 @@ import YumemiWeather
 class WeatherModel {
     weak var delegate: WeatherDelegate? = nil
     
+    let df: DateFormatter = {
```

Comment:
クラス外部に公開しない変数、メソッドはprivateをつけるようにします。
実務で複数人で開発する場合は自分が作成したクラスを他の人が使うことになるため、可視性の設定は気を使います。

```suggestion
    private let df: DateFormatter = {
```